index.rb

```diff
@@ -126,6 +126,24 @@ def show(**kwargs)
   end
 end
 
+venues << Venue.new(:name => 'DNA Lounge', :link => 'https://www.dnalounge.com/calendar/dnalounge.rss') do
```

Comment:
Looks like `link` should be `https://www.dnalounge.com` so that the venue link goes there instead of the rss feed.

Reply:
Oh right. I'll fix that up